slices/tzdata.yaml

```diff
@@ -32,20 +32,227 @@ slices:
     essential:
       - tzdata_config
     contents:
-      /usr/share/zoneinfo/Africa/**:
+      /usr/share/zoneinfo/Africa/Abidjan:
```

Comment:
Unless we use a script to generate this SDF, I think we should stick to using globs to make this more maintainable.

Reply:
As discussed offline, we can't use globs because of conflicts across packages.